pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -71,8 +71,6 @@ func NewStorageFactoryConfig() *StorageFactoryConfig {
 		//
 		// TODO (https://github.com/kubernetes/kubernetes/issues/108451): remove the override in 1.25.
 		// apisstorage.Resource("csistoragecapacities").WithVersion("v1beta1"),
-		admissionregistration.Resource("validatingadmissionpolicies").WithVersion("v1beta1"),
```

Comment:
why did this need to change in this PR?  Not claiming it's wrong, but it stands out since it may be related the storage version change that caught my eye.

Reply:
https://github.com/kubernetes/kubernetes/pull/122891#discussion_r1610713126

If we did not remove this, then VAP would unconditionally be stored at v1beta1 regardless of the set emulation version

Reply:
perhaps a separate PR moving these to floating values so it's smaller and we can assess each one.